### PR TITLE
Fix spelling mistake

### PR DESCRIPTION
**Pull request by kjinan**

### CODE REVIEW

File: tsdb/head.go [hunk 1]
```diff
@@ -1733,7 +1733,7 @@ func (h *Head) Close() error {
 }
 
 // String returns an human readable representation of the TSDB head. It's important to
-// keep this function in order to avoid the struct dump when the head is stringified in
+// keep this function in order to avoid the struct dump when the head is stringfield in
```
Comment (roidelapluie) [inline tsdb/head.go:1736]: stringified is not perfect, but it is the meaning: We could expand that to "converted to string".

stringfield is not correct.

File: discovery/kubernetes/endpoints.go [hunk 2]
```diff
@@ -221,7 +221,7 @@ func (e *Endpoints) buildEndpoints(eps *apiv1.Endpoints) *targetgroup.Group {
 		endpointsNameLabel: lv(eps.Name),
 	}
 	e.addServiceLabels(eps.Namespace, eps.Name, tg)
-	//add endponits labels metadata
+	//add endpoints labels metadata
```
Comment (roidelapluie) [inline discovery/kubernetes/endpoints.go:224]:
```suggestion
	// Add endpoints labels metadata.
```

**Review by roidelapluie — COMMENTED** [review]
Thank you! I have left some suggestions.

File: tsdb/head.go (hunk 1 above)
Comment (roidelapluie) [reply tsdb/head.go:1736]: Just to confirm: stringified is a correct word so I would not change it: https://en.wiktionary.org/wiki/stringified

**Review by roidelapluie — COMMENTED** [review]
stringified is not a typo, can you please remove it from this pull request? Thanks!

**Comment by kjinan** [comment]
> stringified is not a typo, can you please remove it from this pull request? Thanks!

done~

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**